deluge/_libtorrent.py

```diff
@@ -24,8 +24,8 @@
 except ImportError:
     import libtorrent as lt
 
-REQUIRED_VERSION = '1.1.2.0'
-LT_VERSION = lt.__version__
+REQUIRED_VERSION = '1.2.0.0'
+LT_VERSION = lt.version
```

Comment:
This should not be changed, the de-facto standard for getting Python lib version is `__version__`